test/hls/hls_parser_unit.js

```diff
@@ -1935,6 +1935,44 @@ describe('HlsParser', () => {
     await testHlsParser(master, media, manifest);
   });
 
+  it('if unable to guess mime type through Header Request', async () => {
```

Comment:
The name of this test doesn't make sense.  In the previous version, it was inside a describe called "Errors out", leading to the complete name "Errors out if ..."

Here, this might be a better name: `it('falls back to mp4 if HEAD request fails'`

Though we sometimes break this rule a bit, in Jasmine, the "it" functions should read like a sentence about the thing under test: "it falls back to mp4 if ..."

Reply:
Ohh